### PR TITLE
improve unai's text

### PR DESCRIPTION
**Pull request by uorbe001**

### CODE REVIEW

File: partials/team.html [hunk 1]
```diff
@@ -65,7 +65,7 @@ <h4 class="heading">Meet the team</h4>
         "id": "unai",
         "position": "Developer",
         "team": "development",
-        "text": "Unai Orbe graduated from the University of Deusto and has worked for a range of start-ups programming in JS Ruby on Rails. He is currently working on iOS and mobile application development, which is essential to meet Gamevy’s ambitions to create the highest quality experience on mobile devices."
```
Comment (grahamrhay) [inline partials/team.html]: what's "JS ROR"?

Comment (uorbe001) [inline partials/team.html]: that's why I removed it, it makes no sense

File: partials/team.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h4 class="heading">Meet the team</h4>
...
+        "text": "Unai Orbe graduated from the University of Deusto and has worked at a range of start-ups. He has finally been released from the yoke of being Gamevy’s iOS expert and has now returned to HTML5 development and his true love - world domination,"
```
Comment (grahamrhay) [inline partials/team.html]: #trailing_comma

Comment (uorbe001) [inline partials/team.html]: not anymore

**Pull request merged**